apps/rush-lib/src/run.ts

```diff
@@ -0,0 +1,3 @@
+import { start } from './start';
+
+start(false);
```

Comment:
Why do we need this file?

index.ts and start.ts are intended to be special, standardized entry points